docs/source/howto/setup_codes.rst

```diff
@@ -0,0 +1,185 @@
+.. _how-to:setup_code:
+
+*******************
+How to setup a code
+*******************
+
+Once you have at least one computer configured (see :ref:`how-to:setup_computer`), you can configure the codes.
+In AiiDA, for full reproducibility of each calculation, we store each code in the database, and attach to each calculation a given code. This has the further advantage to make very easy to query for all calculations that were run with a given code (for instance because I am looking for phonon calculations, or because I discovered that a specific version had a bug and I want to rerun the calculations).
+
+In AiiDA, we distinguish two types of codes: **remote** codes and **local** codes, where the distinction between the two is described here below.
+
+Local codes
+-----------
+
+With local codes we denote codes for which the code is not already present on the remote machine, and must be copied for every submission.
+This is the case if you have for instance a small, machine-independent Python script that you did not copy previously in all your clusters.
+
+In AiiDA, a local code can be set up by specifying:
+
+* A folder, containing all files to be copied over at every submission
+* The name of executable file among the files inside the folder specified above
+
+Remote codes
+------------
+
+With remote codes we denote codes that are installed/compiled on the remote computer.
+Indeed, this is very often the case for codes installed in supercomputers for high-performance computing applications, because the code is typically installed and optimized on the supercomputer.
```

Comment:
```suggestion
This is usually the case for codes installed in supercomputers for high-performance computing applications, because the code is typically installed and optimized on the supercomputer.
```

Reply:
closing as changed after leo's merges